lib/util/git/error.ts

```diff
@@ -63,7 +64,7 @@ export function checkForPlatformFailure(err: Error): Error | null {
       logger.debug({ err }, 'Converting git error to CONFIG_VALIDATION error');
       const res = new Error(CONFIG_VALIDATION);
       res.validationError = message;
-      res.validationMessage = err.message;
+      res.validationMessage = `\`${err.message.replace(regEx(/`/g), "'")}\``;
```

Comment:
shouldn't we use `replaceAll` supported since node v15
```suggestion
      res.validationMessage = `\`${err.message.replaceAll('`', "'")}\``;
```
so we don't need regex at all 🙃

maybe a string util function makes sense too.

Reply:
i disagree, 
Errors here are coming from try/catch, so they could be anything. 
its not handwritten like the config validation errors